### PR TITLE
Content items admin list

### PR DESCRIPTION
**Pull request by agriffard**
- Use icons instead of text.
- Filter on Content type.
- Badges.

image

### CODE REVIEW

**Comment by hishamco** [comment]
@agriffard seems you read my mind, I just planned yesterday to do an exact PR to use icons instead of buttons, coz the colors confused me and many of us

Great job @agriffard

**Comment by hishamco** [comment]
Perhaps we need to change the "New" button too, I notice there's an issue in the mobile view

File: src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsButtonActions_SummaryAdmin.cshtml [hunk 1]
```diff
@@ -48,5 +48,5 @@
 
 @if (hasClonePermission)
 {
-    <a asp-action="Clone" asp-controller="Admin" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@contentItem.ContentItemId" asp-route-returnUrl="@FullRequestPath" itemprop="UnsafeUrl" class="dropdown-item btn-sm">@T["Clone"]</a>
+    <a asp-action="Clone" asp-controller="Admin" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@contentItem.ContentItemId" asp-route-returnUrl="@FullRequestPath" itemprop="UnsafeUrl" class="dropdown-item btn-sm">@T["Cloneb"]</a>
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentsButtonActions_SummaryAdmin.cshtml:51]: typo in `Cloneb`

File: src/OrchardCore.Themes/TheAdmin/Assets/scss/modules/admin/_list.scss [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+.btn-filter {
+    padding: 0.1rem 0 0.4rem 0 !important;
+}
+
+.button-group {
+    display: none;
+}
+
+li:hover .button-group {
```
Comment (hishamco) [inline src/OrchardCore.Themes/TheAdmin/Assets/scss/modules/admin/_list.scss:13]: I think we could use bootstrap classes `display-inline` and others

**Review by hishamco — CHANGES_REQUESTED** [review]
LGTM after fix my comments

**Comment by agriffard** [comment]
image

**Comment by agriffard** [comment]
See @matiasmolleja's design: https://github.com/OrchardCMS/OrchardCore/pull/2102#issuecomment-408937723

**Pull request merged**